test/scripts/run-gateway-conformance.sh

```diff
@@ -40,7 +40,10 @@ spec:
   controllerName: projectcontour.io/gateway-controller
 EOF
 
+GO_MOD_GATEWAY_API_VERSION=$(grep "sigs.k8s.io/gateway-api" go.mod | awk '{print $2}')
+
 cd $(mktemp -d)
 git clone https://github.com/kubernetes-sigs/gateway-api
 cd gateway-api
+git checkout "${GO_MOD_GATEWAY_API_VERSION}"
 go test ./conformance -gateway-class=contour
```

Comment:
Oof, looks like release-1.21 uses Gateway API v0.4.3: https://github.com/projectcontour/contour/blob/release-1.21/go.mod#L43

Which doesn't have conformance tests: https://github.com/kubernetes-sigs/gateway-api/tree/v0.4.3/conformance

Which I guess was why we were running them from `main` back then.

Probably OK to just drop the gateway-conformance run entirely?

Reply:
yeah I spent a little too long trying to figure out what version would make this work and be what we were using back then, but not super important tbh, so gonna drop it